.coafile

```diff
@@ -7,31 +7,31 @@ ignore =
 max_line_length = 80
 use_spaces = True
 
-[python]
+[all.python]
 # Patches may conflict with autopep8 so putting them in own section so they
 # will be executed sequentially; also we need the LineLengthBear to double
 # check the line length because PEP8Bear sometimes isn't able to correct the
 # linelength.
-bears = SpaceConsistencyBear, PyUnusedCodeBear
+bears = SpaceConsistencyBear, PyUnusedCodeBear, QuotesBear
```

Comment:
Create a new PR for QuotesBear , and exclude tests/* to reduce the size of the PR.

I cant see how this fixes https://github.com/coala/coala-quickstart/issues/62 , maybe because the PR is too noisy.

In general, one PR per issue.

Reply:
make a new commit on the same PR instead? will have to deal with merge-conflicts